src/index.js

```diff
@@ -140,6 +145,28 @@ module.exports = (apiKey, options = {}) => {
     return request('POST', 'search', params);
   };
 
+  /**
+   * PlaceKit reverse geocoding
+   * @memberof client
+   * @param {string} coordinates Coordinates "lat,lng"
+   * @param {Options} [opts] Override global parameters
+   * @return {Promise<SearchResponse>}
+   */
+  client.reverse = (coordinates, opts = {}) => {
+    if (!['string', 'undefined'].includes(typeof coordinates)) {
```

Comment:
Same comment as above, API's `coordinates` param is optional because we can still locate pretty well with just the IP.